extension/js/background.js

```diff
@@ -72,9 +35,10 @@ async function makeRelayAddressForTargetElement(info, tab) {
     return;
   }
 
-  browser.tabs.executeScript(tab.id, {
-    frameId: info.frameId,
-    code: escaped`let inputElement = browser.menus.getTargetElement(${info.targetElementId});inputElement.value = "${newRelayAddressResponse}";`,
+  browser.tabs.sendMessage(tab.id, {
```

Comment:
`Error: Could not establish connection. Receiving end does not exist.`

Reply:
Fixed in 155045b.